packages/forma-36-react-components/src/components/ControlledInput/ControlledInput.css

```diff
@@ -10,3 +11,139 @@
 .ControlledInput--disabled {
   cursor: not-allowed;
 }
+
+.RadioButton {
+  display: inline-block;
+  position: relative;
+  text-overflow: ellipsis;
+  line-height: var(--spacing-l);
+  min-height: var(--spacing-l);
+  margin: 0;
+  width: var(--spacing-m);
+  height: var(--spacing-m);
+  min-height: var(--spacing-m);
```

Comment:
I'm not sure if I understand this correctly, but should we use spacing tokens for the width and height of things?

Reply:
@gui-santos Valid point. I wanted to get it from tokens, instead of hardcoding, maybe we could export additional tokens for those purposes? Components like `Icon` would also benefit from it. But maybe you are right, for now maybe I should hardcode it, like it is done in Icon for example.